game/src/api/systems/PlayerSystem.java

```diff
@@ -1,24 +1,25 @@
-package ecs.systems;
-
+package api.systems;
+
+import api.Entity;
+import api.Game;
+import api.System;
+import api.components.PlayerComponent;
+import api.components.VelocityComponent;
+import api.utils.componentUtils.MissingComponentException;
 import com.badlogic.gdx.Gdx;
-import configuration.KeyboardConfig;
-import ecs.components.MissingComponentException;
-import ecs.components.PlayableComponent;
-import ecs.components.VelocityComponent;
-import ecs.entities.Entity;
-import ecs.tools.interaction.InteractionTool;
-import starter.Game;
+import content.configuration.KeyboardConfig;
```

Comment:
hmm, das hab ich komplett übersehen. 
Muss ich nochmal grübeln.

Reply:
Im neuen Entwurf ist das alles content, in der api sollten keine expliziten Tastendrücke implementiert sein